packages/server/src/migrations/schema/v69.ts

```diff
@@ -0,0 +1,374 @@
+  await client.query('ALTER TABLE IF EXISTS "StructureMap" DROP COLUMN IF EXISTS "contextTypeQuantity"');
+  await client.query('ALTER TABLE IF EXISTS "StructureMap" DROP COLUMN IF EXISTS "contextTypeValue"');
+  await client.query('ALTER TABLE IF EXISTS "Subscription" DROP COLUMN IF EXISTS "contact"');
+  await client.query('ALTER TABLE IF EXISTS "Substance" DROP COLUMN IF EXISTS "category"');
+  await client.query('ALTER TABLE IF EXISTS "Substance" DROP COLUMN IF EXISTS "code"');
+  await client.query('ALTER TABLE IF EXISTS "Substance" DROP COLUMN IF EXISTS "containerIdentifier"');
+  await client.query('ALTER TABLE IF EXISTS "SubstanceSpecification" DROP COLUMN IF EXISTS "code"');
+  await client.query('ALTER TABLE IF EXISTS "SupplyRequest" DROP COLUMN IF EXISTS "category"');
+  await client.query('ALTER TABLE IF EXISTS "Task" DROP COLUMN IF EXISTS "businessStatus"');
+  await client.query('ALTER TABLE IF EXISTS "Task" DROP COLUMN IF EXISTS "code"');
+  await client.query('ALTER TABLE IF EXISTS "Task" DROP COLUMN IF EXISTS "groupIdentifier"');
+  await client.query('ALTER TABLE IF EXISTS "Task" DROP COLUMN IF EXISTS "performer"');
+  await client.query('ALTER TABLE IF EXISTS "TerminologyCapabilities" DROP COLUMN IF EXISTS "contextType"');
+  await client.query('ALTER TABLE IF EXISTS "TerminologyCapabilities" DROP COLUMN IF EXISTS "jurisdiction"');
+  await client.query('ALTER TABLE IF EXISTS "TerminologyCapabilities" DROP COLUMN IF EXISTS "contextTypeQuantity"');
+  await client.query('ALTER TABLE IF EXISTS "TerminologyCapabilities" DROP COLUMN IF EXISTS "contextTypeValue"');
+  await client.query('ALTER TABLE IF EXISTS "TestScript" DROP COLUMN IF EXISTS "contextType"');
+  await client.query('ALTER TABLE IF EXISTS "TestScript" DROP COLUMN IF EXISTS "jurisdiction"');
+  await client.query('ALTER TABLE IF EXISTS "TestScript" DROP COLUMN IF EXISTS "contextTypeQuantity"');
+  await client.query('ALTER TABLE IF EXISTS "TestScript" DROP COLUMN IF EXISTS "contextTypeValue"');
+  await client.query('ALTER TABLE IF EXISTS "ValueSet" DROP COLUMN IF EXISTS "contextType"');
+  await client.query('ALTER TABLE IF EXISTS "ValueSet" DROP COLUMN IF EXISTS "jurisdiction"');
+  await client.query('ALTER TABLE IF EXISTS "ValueSet" DROP COLUMN IF EXISTS "contextTypeQuantity"');
+  await client.query('ALTER TABLE IF EXISTS "ValueSet" DROP COLUMN IF EXISTS "contextTypeValue"');
+  await client.query('ALTER TABLE IF EXISTS "UserSecurityRequest_Token" DROP COLUMN IF EXISTS "index"');
+  await client.query('ALTER TABLE IF EXISTS "AsyncJob" DROP COLUMN IF EXISTS "status"');
+  await client.query('ALTER TABLE IF EXISTS "AsyncJob" DROP COLUMN IF EXISTS "user"');
+  await client.query('ALTER TABLE IF EXISTS "ContactPoint" DROP COLUMN IF EXISTS "id"');
+  await client.query('ALTER TABLE IF EXISTS "ContactPoint" DROP COLUMN IF EXISTS "index"');
+  await client.query('ALTER TABLE IF EXISTS "ContactPoint" DROP COLUMN IF EXISTS "content"');
```

Comment:
Should we drop the `ContactPoint` and `Identifier` tables entirely?

Reply:
Yeah, probably.  I can tackle that in a separate PR